docker-compose.yml

```diff
@@ -38,3 +46,11 @@ services:
     image: 'shafreen/db-setup:0.0.1'
     network_mode: "host"
 
+  nats:
+    image: nats:latest
+    command: "-c /nats-server.conf"
+    volumes:
+      - "./nats-server.conf:/nats-server.conf"
```

Comment:
Can't we just copy the file just like we copy Config.toml?

Reply:
This configuration file is for `nats-server` not for `sms-sender-service`. Hence, we can't use `Cloud.toml` for this.

Reply:
You don't really need this entire config file at all for this. This was added to nats tests to increase the number of connections with the server.

Reply:
```
    nats-1:
        image: nats:latest
        hostname: foo
        ports:
            - "4222:4222"
            - "8222:8222"
```
will do

Reply:
Everything else LGTM.

Reply:
Nice, thanks :+1:

Reply:
Fixed with [065b0b2be75f57bc8c6109b70abb9b60c64a43ad](https://github.com/shafreenAnfar/ballerina-social-media/pull/7/commits/065b0b2be75f57bc8c6109b70abb9b60c64a43ad)